scripts/pypipublish.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/zsh
+# pypipublish.sh
+# Usage: Run ./scripts/pypipublish.sh path/to/project/root conda-env-name
```

Comment:
Wouldn't it make more sense to create an empty environment rather than depend on an existing one?